### ISSUE TITLE
ProviderLocalRefresh doesn't store the cookie for the auth token

### ISSUE THREAD

**Issue by mlorenzog** [sidebase/nuxt-auth#786]
### Environment

- Operating System: `Darwin`
- Node Version:     `v18.17.1`
- Nuxt Version:     `3.11.0`
- CLI Version:      `3.10.1`
- Nitro Version:    `2.9.4`
- Package Manager:  `yarn@1.22.21`
- Builder:          `-`
- User Config:      `devtools`, `modules`, `srcDir`, `image`, `i18n`, `colorMode`, `app`, `components`, `runtimeConfig`, `auth`
- Runtime Modules:  `@nuxtjs/eslint-module@4.1.0`, `@nuxt/ui@2.15.2`, `@nuxt/image@1.3.0`, `@nuxtjs/color-mode@3.4.0`, `@nuxtjs/i18n@8.2.0`, `@pinia/nuxt@0.5.1`, `@sidebase/nuxt-auth@0.7.1`
- Build Modules:    `-`

### Reproduction

Start a new project, using this auth configuration:

  auth: {
    globalAppMiddleware: {
      isEnabled: true
    },
    // Important, this variable is set at build time
    baseURL: process.env.AUTH_BASE_URL
    // https://sidebase.io/nuxt-auth/configuration/nuxt-config
    provider: {
      type: 'refresh',
      pages: {
        login: '/auth/login'
      },
      endpoints: {
        signIn: { path: '/api/login', method: 'post' },
        getSession: { path: '/api/v1/user/data', method: 'get' }
        // signOut: { path: '/logout', method: 'post' },
        // signUp: { path: '/register', method: 'post' },
      },

      refreshToken: {
        signInResponseRefreshTokenPointer: '/refresh_token'
      },
      token: {
        signInResponseTokenPointer: '/token'
      }
    }
  }
  
  

### Describe the bug

Using that configuration only the refresh token is stored at the cookie, not the session token. 

We have tested it and detected that changing the same-site attribute of the token cookie to "lax" it works, but by default the same-site attribute is setted as "none" for that schema. According to the source code the default value should be "lax" [(https://github.com/sidebase/nuxt-auth/blob/cb3db19563f8cb3af6dd63097e91305697c3ce8a/src/runtime/types.ts#L168)](https://github.com/sidebase/nuxt-auth/blob/cb3db19563f8cb3af6dd63097e91305697c3ce8a/src/runtime/types.ts#L168)

**Comment by agracia-foticos** [comment]
Related??? https://github.com/sidebase/nuxt-auth/issues/790

**Comment by phoenix-ru** [comment]
@agracia-foticos I don't think it is related, as #790 refers to `authjs` provider, while this refers to `refresh` provider
